src/main/java/woojkk/fintechProject/service/AccountService.java

```diff
@@ -63,9 +68,71 @@ private void validateCreateAccount(AccountUser accountUser, AccountType accountT
     Integer accountTypeCount =
         accountRepository.countByAccountUserAndAccountTypeAndBank(accountUser, accountType, bank);
 
+    Integer unregisteredAccountCount = accountRepository.countByAccountUserAndAccountTypeAndBankAndAccountStatus(
+        accountUser, accountType, bank, AccountStatus.UNREGISTERED);
+
+    accountTypeCount -= unregisteredAccountCount;
```

Comment:
```suggestion
    final Integer registeredAccountTypeCount = accountTypeCount - unregisteredAccountCount;
    
    if (registeredAccountTypeCount >= 5) ...
```
위와 같이 새로운 변수와 네이밍을 지어주는것이 좋습니다. 지역변수를 `final` 로 선언할 수 있도록 지켜보시면 좋을 것 같아요.

변수가 마음대로 변하는 것을 제어할 수 있을 뿐더러 사이드 이펙을 줄일 수 있습니다.